CONTRIBUTING.rst

```diff
@@ -522,11 +522,30 @@ this:
 
 ::
 
-   python -m nox -e "pytest-3.7(coverage=False)" -- tests/unit/cli/test_batch.py; espeak "Tests done, woohoo!"
+    python -m nox -e "test-3(coverage=False)" -- tests/unit/cli/test_batch.py; espeak "Tests done, woohoo!"
 
 That way you don't have to keep monitoring the actual test run.
 
 
+::
+
+    python -m nox -e "test-3(coverage=False)" -- --core-tests
+
+You can enable or disable test groups locally by passing there respected flag:
```

Comment:
Minor grammar fix: "there" should be "their." But actually I'd recommend changing "there" to "these" or "the."